core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -121,6 +121,7 @@
           <a ng-click="closeSubmenuIfNotMobile($event)" href="/learner_dashboard"
              class="protractor-test-dashboard-link protractor-test-learner-dashboard-link">
             <span translate="I18N_TOPNAV_LEARNER_DASHBOARD"></span>
+            <span><[isAdmin]> | <[isTopicManager]> | <[newStructuresEnabled]></span>
```

Comment:
I am not sure what this is intended for, was it meant for debugging? In which case, can you remove this?